executor/runcexecutor/executor.go

```diff
@@ -36,6 +37,8 @@ type Opt struct {
 	CommandCandidates []string
 	// without root privileges (has nothing to do with Opt.Root directory)
 	Rootless bool
+	// CgroupParent is the cgroup-parent name from dockerd
+	CgroupParent string
```

Comment:
`DefaultCgroupParent` as I don't see an issue of defining this per operation as well in the future.

Reply:
done